src/game_timer.cpp

```diff
@@ -0,0 +1,52 @@
+// Header
+#include "game_timer.hpp"
+
+// stlib
+#include <vector>
+#include <algorithm>
+#include <sstream>
```

Comment:
this is hella nitpicky, but can we put all our `#include`s in the `.hpp` file?

Reply:
Bring on the nits!